proto/dymension/rollapp/state_status.proto

```diff
@@ -13,4 +13,6 @@ enum StateStatus {
   STATE_STATUS_RECEIVED = 1; 
   // STATE_STATUS_FINALIZED defines a rollapp state where the the "Dispute Period" has ended and this state is considered final
   STATE_STATUS_FINALIZED = 2; 
+  // STATE_STATUS_DISPUTED defines a rollapp state where the state update transaction was disputed by a participant
+  STATE_STATUS_DISPUTED = 3;
```

Comment:
In common.type we call it "REVERTED". I think we should keep the same terminology. (and also possibly change the RECEIVED to PENDING).

I'd just use the commontypes.STATUS for all the statuses